packages/cli/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Updated the `zowe auth login apiml` command to place its retrieved token into the current base profile instead of creating a new base profile. [#2181](https://github.com/zowe/zowe-cli/issues/2181) & [#1650](https://github.com/zowe/zowe-cli/issues/1650)
+  - The update also places a directive to use token authentication for the default `zosmf` profile if that profile is configured to connect to API-ML.
```

Comment:
I know it'd clear to most people but, I'm curious if we can be a bit more explicit here...

> the current base profile

I do believe this means the `default` base profile. Not to be confused with any `type: base` profiles in the nesting hierarchy of a zosmf profile (e.g. myBase.anotherBase.zosmf vs myDefaultBase).

Reply:
It could be the default base profile, but it is not limited to the default base profile. A user could specify the --base-profile option on the login command and thus specify a base profile which is not the default. I used the term "current" in an attempt to generically refer to the base profile currently requested by the user. I am not opposed to changing this wording if someone can identify a simple phrase to encapsulate the possibilities available to the user.

Reply:
I changed the wording slightly.